bot.user.js

```diff
@@ -7,13 +7,47 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // ==UserScript==
 // @name         Slither.io-bot
 // @namespace    http://slither.io/
-// @version      1.2.3
+// @version      Persistence dev (branched from 1.2.3)
 // @description  Slither.io bot
 // @author       Ermiya Eskandary & Théophile Cailliau
 // @match        http://slither.io/
 // @grant        none
 // ==/UserScript==
 
+/*
+Override bot options here
+Uncomment variables you wish to change from their default values
+Changes you make here will be kept between script versions
+*/
+var customBotOptions = {
```

Comment:
I think this is an error message due to Hound being configured slightly wrong.

@ErmiyaEskandary ? The keyword spacing rule looks for violations like `}else {`. This seems busted.